llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -174,6 +174,18 @@ static cl::opt<std::string> WorkloadDefinitions(
              "}"),
     cl::Hidden);
 
+static cl::opt<bool> ImportAssumeUniqueLocal(
+    "import-assume-unique-local", cl::init(false),
+    cl::desc(
+        "By default, a local-linkage global variable won't be imported in the "
```

Comment:
can the local var be promoted to global to enable importing?

Reply:
As shown in the updated test, with this option on, when the local-var gets imported, it will have a global name `<vtable>.llvm.<mod-hash>`.